File: Corte1/Java/Saludar/Persona.java
package persona;

public class Persona {
    public String nombre;
    public int edad;
    
    public void saludar(){
        System.out.println("Hola soy " + nombre + " y tengo " + edad + " años.");
    }

    public static void main(String[] args) {
        Persona p1 = new Persona();
        Persona p2 = new Persona();
        Persona p3 = new Persona();
        
        p1.nombre = "Felix";
        p1.edad = 19;
        p1.saludar();
        
        p2.nombre = "Steven";
        p2.edad = 18;
        p2.saludar();
        
        p3.nombre = "Capi";
        p3.edad = 18;
        p3.saludar();
    }
    
}

File: Corte1/Python/Persona/Persona.py
class Persona:
    #constructor
    def __init__(self): 
        self.nombre = ""
        self.años = 0
        self.lugar = ""

    #Metodo
    def say_hello(self):
        print(f"Hola, me llamo {self.nombre} , tengo {self.años} año y soy de {self.lugar}.")

def main ():
    #Crea un objeto
    p1 = Persona()

    #Asigna atributos
    p1.nombre = "Felix"
    p1.años = 19
    p1.lugar = "Bogotá"

    #Llama el metodo
    p1.say_hello()

if __name__=="__main__":
    main()